action-scripts/brownie/scripts/report_gauges.py

```diff
@@ -425,12 +427,17 @@ def _parse_added_transaction(transaction: dict, **kwargs) -> Optional[dict]:
     elif isinstance(to_name, str):
         to_string = f"!!f{to_name}??"
 
+    chain = chain.replace("-main", "") if chain else "mainnet"
+    is_preferential = (
+        BalPoolsGauges(chain).get_preferential_gauge(pool_id) == gauge_address
+    )
+
     return {
         "function": f"{to_string}/{command}",
-        "chain": chain.replace("-main", "") if chain else "mainnet",
+        "chain": chain,
         "pool_id_and_address": f"{pool_id} \npool_address: {pool_address}",
         "symbol_and_info": f"{pool_symbol} \nfee: {fee}\na-factor: {a_factor}",
-        "gauge_address_and_info": f"{gauge_address}\nStyle: {style}\ncap: {gauge_cap}",
+        "gauge_address_and_info": f"{gauge_address}\nstyle: {style}\ncap: {gauge_cap}\npreferential: {is_preferential}",
```

Comment:
In other cases where there is an error state we are loud about it
!!NOT PREFERENTIAL!!

It's optional, but seems like it would be easy to implement and helps catch bugs.  In a sense could even not show it if there is no problem, but I think we like seeing what is being reported on each time.

Reply:
wdym error? you mean in the case of the gauge not being preferential?